lib/ansible/modules/cloud/amazon/aws_kms.py

```diff
@@ -230,16 +234,17 @@ def assert_policy_shape(policy):
         raise Exception('Problems asserting policy shape. Cowardly refusing to modify it: {}'.format(' '.join(errors)))
     return None
 
+
 def main():
     argument_spec = ansible.module_utils.ec2.ec2_argument_spec()
     argument_spec.update(dict(
-        mode = dict(choices=['grant', 'deny'], default='grant'),
-        key_alias = dict(required=False, type='str'),
-        key_arn = dict(required=False, type='str'),
-        role_name = dict(required=False, type='str'),
-        role_arn = dict(required=False, type='str'),
-        grant_types = dict(required=False, type='list'),
-        clean_invalid_entries = dict(type='bool', default=True),
+        mode=dict(choices=['grant', 'deny'], default='grant'),
+        key_alias=dict(required=False, type='str'),
```

Comment:
You don't need `required=False`, it's the default. And we tend to start with the parameter type.